### ISSUE TITLE
Contributors beeing blocked

### ISSUE THREAD

**Issue by scratchpads** [NaturalHistoryMuseum/scratchpads2#6392]
- Added by [Jakovos Demetriou](http://alientoma.myspecies.info/el/user/5)
- Scratchpad: [Alientoma](http://alientoma.myspecies.info)

### Description:

Some contributors submitting observational data of species are being blocked and cannot fill the data form placed on the website.

How can these people be unblocked?

**Comment by scratchpads** [comment]
Comment by [Jakovos Demetriou](https://alientoma.myspecies.info/el/user/5)

Form link: https://alientoma.myspecies.info/el/node/233

Message people see: "You have been blocked. You are unable to access this website"

**Comment by therobyouknow** [comment]
Thank you for advising on the error and location of where it as been seen. I have started investigating.

 "You have been blocked. You are unable to access this website" does not appear to be in the scratchpads code base on master branch. 

My next steps would be:
- to determine the git repo branch of scratchpads used for your site, if different branch from the master branch I've searched. Thinking here is that the message may be in scratchpad code that is in a branch that your production site is using which has code for showing this message which the master branch does not have. I would find this out by checking the deployment steps for your site and/or branch active on the production server.
- Also another plausible reason for the message is that it's not part of scratchpads but coming from the infrastructure - e.g. a firewall or web server

Questions/Requests I have:
- Can you paste in a screenshot of the page that shows the error? this can help me determine if it is something in the site code or infrastructure, depending on the appearance of the page
- Is it all contributors or just some that can't access? 
- when those contributors are locked out/blocked, can other contributors still access the page? This might indicate that I would need to investigate the issue with infrastructure rather than the site code
- any further information about the contributors who were blocked/locked out in terms of geographical location/country and if they are running VPN software
- have they been able to overcome being blocked by any means, e.g. my resetting their internet connection, or using another service to get a different IP (by DHCP, power cycling their router). Not suggesting this is the ultimate solution, but may promote the idea the issue is a firewall one

**Comment by Archilegt** [comment]
@therobyouknow, this may be an error coming from the time (fairly recent but before you arrived) when Scratchpads was moved to https and the firewall security increased. People may not be being blocked any longer. I was being blocked myself often, but not any longer. Also, it should be possible to add the users' IP to exception lists.

**Comment by benscott** [comment]
Yes, this issue was moved by the Scratchpads move behind the Museum firewall and the way the Scratchpads handled the SSL termination, which should all be fixed now.
